Add `info` command (#993)
paritytech/cargo-contract: docs/info.md

# Other commands
`cargo-contract` provides CLI support for displaying info directly from the command
line.

### `info`

> Fetch and display informations for a given contractId (AccountId). 

Reviewer: ```suggestion
Fetch and display information for a given contract.
```

> Display these information (https://github.com/paritytech/substrate/blob/master/frame/contracts/src/storage.rs#L45-L69) from 
> 
> 
> e.g. `cargo contract info --suri //Alice`
> 
> Assumes that `cargo contract build`, `cargo contract upload` and `cargo contract instantiate` have already been run to produce the contract artifacts.
> 

Reviewer: ```suggestion
```

This is not correct, also `--suri` is not necessary for `cargo contract info`.

```
cargo contract info \
       --contract 5DVGLfDGBvqMr9nCg48g99oD8Mz3sruWmb6ek5UbWvDnbTgZ
```

> - `--contract` the account id of the contract to invoke, returned after a successful `contract instantiate`

Author: What is the purpose of invoking `cargo contract info` if the contract is instantiated ?

> - `--contract` the account id of the contract to invoke, returned after a successful `contract instantiate`

Reviewer: I removed

>returned after a successful `contract instantiate`

because doing `cargo contract instantiate` is not needed to be done before `cargo contract info`. A user can also obtain a contract id via some other mean.

> - `--contract` the account id of the contract to invoke, returned after a successful `contract instantiate`

Author: Yes I agree on the fact that the user can get the account id in other way but if these commands are not run, no information will be displayed.

> - `--contract` the account id of the contract to invoke, returned after a successful `contract instantiate`

Reviewer: Executing those commands is not the only way to deploy a contract, users can also use e.g. the Contracts UI, `polkadot-js` or Swanky.

*Optional*

> - `--url` the url of the rpc endpoint you want to specify - by default ws://localhost:9944 .

Reviewer: ```suggestion
- `--url` the url of the rpc endpoint you want to specify - by default `ws://localhost:9944`.
```
